prov/gni/src/gnix_cntr.c

```diff
@@ -296,6 +296,36 @@ DIRECT_FN STATIC int gnix_cntr_wait(struct fid_cntr *cntr, uint64_t threshold,
 	return gnix_cntr_wait_sleep(cntr_priv, threshold, timeout);
 }
 
+DIRECT_FN STATIC int gnix_cntr_adderr(struct fid_cntr *cntr, uint64_t value)
+{
+	struct gnix_fid_cntr *cntr_priv;
+
+	cntr_priv = container_of(cntr, struct gnix_fid_cntr, cntr_fid);
+	atomic_add(&cntr_priv->cnt_err, (int)value);
+
+	if (cntr_priv->wait)
+		_gnix_signal_wait_obj(cntr_priv->wait);
+
+	//TODO: _gnix_trigger_check_err_cntr(cntr_priv);
```

Comment:
Remove these comment lines unless it is determined that they want to add that functionality.